src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -271,7 +294,10 @@ public int getTtl() {
     }
 
     private void postInsertActions(String aggregatedObject, RulesObject rulesObject, String event, String id) {
-        eventToObjectMap.updateEventToObjectMapInMemoryDB(rulesObject, event, id);
-        subscriptionHandler.checkSubscriptionForObject(aggregatedObject, id);
+        eventToObjectMap.updateEventToObjectMapInMemoryDB(rulesObject, event, id, getTtl());
+    }
+    
+    public void checkAggregations(String aggregatedObject, String id) {
+    	subscriptionHandler.checkSubscriptionForObject(aggregatedObject, id );
```

Comment:
extra space after last parameter

Reply:
Also, missing java doc for this new public method.

Reply:
done